packages/teams-js/src/public/meeting.ts

```diff
@@ -925,6 +926,47 @@ export namespace meeting {
     );
   }
 
+  /**
+   * This function is used to join a meeting
+   * @param joinMeetingParams
+   */
+  export function joinMeeting(joinMeetingParams: JoinMeetingParams): Promise<string | object> {
```

Comment:
Please add an `@returns` section to your function comment detailing what this function returns and how developers should use that return value.

Reply:
Actually, why does this return anything at all other than `Promise<void>`? It looks like you're not calling a function that returns a value inside of the promise resolution

Reply:
@TrevorJoelHarris, changed the return type to Promise<void>